src/pages/Dashboard/Portfolio.tsx

```diff
@@ -34,7 +36,16 @@ export const Portfolio = () => {
 
   const assetIds = useAppSelector(selectPortfolioAssetIds)
 
-  const totalBalance = useAppSelector(selectPortfolioTotalFiatBalanceIncludingStaking)
+  // TODO(gomes): This goes away in a follow-up PR
```

Comment:
Pretty much what this comment says - now that Cosmos validatorDataSlice and portfolio validator data is gone, what we call `includeStaking` is misleading - it really doesn't include *all* staking data because something something reward denoms.

This brings a more accurate way, relying *purely* on fiat balances (which don't have the denom issue stated above) and earn balances/portfolio balances being separate. Note the *excludeEarnDupes* in the selector name which does precisely this, exclude balances which are also present in earn, so we don't double count them.